plugins/sharding/logical_select.rb

```diff
@@ -152,6 +152,11 @@ def write_records(writer, context)
             :offset => current_offset,
             :limit => current_limit,
           }
+          context.sort_keys.each do |sort_key|
+            if sort_key.match("-timestamp")
```

Comment:
We shouldn't assume `timestamp` as sort key.
It should be `sort_key.start_with?("-")`.

Reply:
If `sort_keys` have two `-XXX`, `result_sets` isn't reversed.
It should be:

```ruby
if context.sort_keys.any? {|sort_key| sort_key.start_with?("-")}
  result_sets = result_sets.reverse
end
```

or something.